src/server/api/lib/message-sending.js

```diff
@@ -18,16 +18,15 @@ export async function getLastMessage({ contactNumber, service }) {
 
 export async function saveNewIncomingMessage(messageInstance) {
   if (messageInstance.service_id) {
-    const countResult = await r.getCount(
-      r.knex("message").where("service_id", messageInstance.service_id)
-    );
+    const [countResult] = await r
+      .knex("message")
+      .where("service_id", messageInstance.service_id)
+      .select("id")
+      .limit(1);
     if (countResult) {
-      console.error(
-        "DUPLICATE MESSAGE SAVED",
-        countResult.count,
-        messageInstance
-      );
+      console.error("DUPLICATE MESSAGE", countResult, messageInstance);
     }
+    return;
```

Comment:
Can you explain the `return` here? Is this meant to be inside the `if (countResult) {` block?

Reply:
dur, the `return` should have been inside the if-clause.  I've moved it inside and also renamed the variable duplicateMessage to be clearer.